README.md

```
@@ -216,10 +216,10 @@ See [LICENSE] for details.
 [update page]: https://affine.pro/blog?tag=Release%20Note
 [jobs available]: ./docs/jobs.md
 [latest packages]: https://github.com/toeverything/AFFiNE/pkgs/container/affine-self-hosted
-[contributor license agreement]: https://github.com/toeverything/affine/edit/canary/.github/CLA.md
+[contributor license agreement]: https://cla-assistant.io/toeverything/AFFiNE
 [stars-icon]: https://img.shields.io/github/stars/toeverything/AFFiNE.svg?style=flat&logo=github&colorB=red&label=stars
 [codecov]: https://codecov.io/gh/toeverything/affine/branch/canary/graphs/badge.svg?branch=canary
-[node-version-icon]: https://img.shields.io/badge/node-%3E=18.16.1-success
+[node-version-icon]: https://img.shields.io/badge/node-%3E=22.12.0-success
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "node-version-icon"

(MD053, link-image-reference-definitions)